roles/st2/tasks/datastore.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Add the st2 system service user to the service group
+  become: yes
+  user:
+    user: "{{ st2_system_service_user }}"
+    groups: "{{ st2_system_service_group }}"
+    append: yes
+
+- name: Make stackstorm encryption keys directory
+  become: yes
+  file:
+    path: "{{ st2_datastore_key_file | dirname }}"
+    state: directory
+    mode: 0750
+    owner: root
+    group: "{{ st2_system_service_group }}"
```

Comment:
Just hardcoding to `st2` would be sufficient, because there is no other option for now (st2 service user is not configurable)